src/main/kotlin/app/utils/CommandRemove.kt

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.utils
+
+import com.beust.jcommander.Parameter
+import com.beust.jcommander.Parameters
+
+@Parameters(separators = "=",
+        commandDescription = "Remove specific repository from tracked list")
+class CommandRemove {
+    // Path to analyzed repository.
+    @Parameter(description = "REPOPATH")
```

Comment:
It is used in Main via jc.usage() method that displays usage options based on annotated descriptions.